test/static-export-test/library.cc

```diff
@@ -0,0 +1,7 @@
+#include <fmt/compile.h>
+
+__attribute__((visibility("default")))
+std::string foo()
+{
+    return fmt::format(FMT_COMPILE("foo bar {}"), 4242);
+}
```

Comment:
Please apply clang-format here and below.

Reply:
Oh! I completely forgot about clang-format.